### PR TITLE
Enable/disable JSON RPC transaction APIs with ENABLE_JSON_RPC_TX_API flag

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Enable/disable JSON RPC transaction APIs with ENABLE_JSON_RPC_TX_API flag
- Set NUM_GENESIS_ACCOUNTS=10
- Fix testnet genesis params file 

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/680

### CODE REVIEW

File: json_rpc/index.js [hunk 1]
```diff
@@ -29,472 +29,481 @@ const PathUtil = require('../common/path-util');
...
+      trafficStatsManager.addEvent(TrafficEventTypes.JSON_RPC_SET);
+      let result = false;
+      if (await p2pServer.node.injectAccountFromHDWallet(args.encryptedMnemonic, args.index)) {
+        result = true;
+        p2pServer.client.run();
+      }
+      return addProtocolVersion({ result });
+    },
+
+    // TODO(minsulee2): Add p2p json rpc API methods here.
```
Comment (liayoo) [inline json_rpc/index.js:409]: should `p2p_getRouteStatus` be added under get methods or another group (e.g. p2p methods?)
cc: @minsulee2

Comment (platfowner) [reply json_rpc/index.js:409]: It's in `const systemSetMethods =`, which is always enabled regardless of ENABLE_JSON_API_APP_SET_API flag value.

Comment (liayoo) [reply json_rpc/index.js:409]: I think the naming (systemSetMethods) threw me off, since p2p_getRouteStatus seems to be a get API?

Comment (platfowner) [reply json_rpc/index.js:406]: Right, it was not clear enough. Tried to apply two groups: non-tx methods and tx methods with new flag name: ENABLE_JSON_RPC_TX_API.
PTAL~ @liayoo

Comment (liayoo) [reply json_rpc/index.js:406]: Thanks for the clarification! Looks good!

Comment (liayoo) [reply json_rpc/index.js:406]: + we should discuss which client & json-rpc APIs to enable on the API servers~

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by platfowner** [comment]
Thanks for the review, now merging..

**Pull request merged**